src/main/drivers/pwm_output.c

```diff
@@ -135,7 +136,9 @@ static void pwmWriteMultiShot(uint8_t index, uint16_t value)
 
 void pwmWriteMotor(uint8_t index, uint16_t value)
 {
-    pwmWritePtr(index, value);
+    if (motors[index].enabled) {
+        pwmWritePtr(index, value);
+    }
```

Comment:
or setting the pwmWritePtr to a bogus function, that does nothing ... if there are not enough motors enabled to support the mixer.